packages/server/graphql/public/types/Organization.ts

```diff
@@ -14,6 +14,9 @@ const Organization: OrganizationResolvers = {
   company: async ({activeDomain}, _args, {authToken}) => {
     if (!activeDomain || !isSuperUser(authToken)) return null
     return {id: activeDomain}
+  },
+  featureFlags: ({featureFlags}) => {
```

Comment:
Why do we need to add this? What does it do?

Reply:
it exposes `featureFlags` on organization. So we can query

```
 organization {
  featureFlags {
    teamsLimit
  }
}
```

Reply:
@nickoferrall I just followed same pattern we use for the User, I saw we moved featureFlags from just types to public types: https://github.com/ParabolInc/parabol/pull/6664/files#diff-020fe869f7d8e452f6fc5eee264fd6082069b29a2c13cb182230a972e7da8872L84-L94

I guess it is just a new pattern to define types? I specify it in `public/typeDefs` then write resolver like this.